referenceqvm/unitary_generator.py

```diff
@@ -187,16 +190,17 @@ def permutation_arbitrary(args, num_qubits):
         start_i - starting index to lift gate from
     :rtype:  tuple (sparse_array, np.array, int)
     """
-    # Check input
-    if type(args) is tuple or type(args) is list:
-        if len(args) is 0:
-            raise TypeError("Need at least one qubit index to perform"
-                            "permutation")
-    else:
+    # Don't permit NoneType or empty sequences, but allow 0
```

Comment:
we need to adjust the docstring parameter based on this.

Reply:
Can do.